control/modelsimp.py

```diff
@@ -211,26 +218,46 @@ def balred(sys, orders, method='truncate'):
         of systems)
     method: string
         Method of removing states, either ``'truncate'`` or ``'matchdc'``.
+    alpha: float
+        Specifies the alpha-stability boundary for the eigenvalues
```

Comment:
The original text from SLICOT to which I am referring is in [lines 89-96 of AB09MD.f](https://github.com/python-control/Slycot/blob/b01430c0f5117e1d6b6fd1717e04fe765a37ebf2/slycot/src/AB09MD.f#L89-L96)